### DISCUSSION TITLE
Starboard

### DISCUSSION THREAD

**Discussion by twoxfh** [rhashimoto/wa-sqlite#31] [Show and tell]
For what its worth I made a starboard with the demo code at [wa-sqlite demo](https://twoxfh.github.io/starboard-editor/?notebook=bw1%40G5oIAByFcZvThDY8qARN7PbNQN0aJob9XpOgnnff0NbzVQFKwUiHK4Wa8HU30eUIhG9.-ZraJAVScSVyZ1xZyty.3X8BBlcg0p0aG6EK7F2P8DYqHfZQ.danJzg7w0KLnFXFu5RvQyhl4NgWdDwx6Hhh0BOLWvU3L6sfOAx.LyMbULHT0hvA2k6sf7PCbUWZF6z6oSDcpJdHN7y9u-Qu1grnL.5Z6Os.C9NVpnCy-seJmUuTovKj36KjmyrgWU91InwqBurJFHo1WTnJeWKYUA81nFNZsJ2shreZA6a5NdsJVt-usBacTGGkMdpWz5ODhZ1WWKGUoXeOvby3LqOj6LhzVjosuoBUN0dQ9tZGaxhma3Nu2pdMwxw80wRH9Ey..tJDkh7r5Rm9Dm7Z0ELJbhrP1AYqJmdaG9zOfAb5jA837BEFuU5unUjMIh4WWr5Gli5hTUfyKX1ZalMw8It0GTnpAwKcc7RZ6pAAFbqz-nbm98HNTBj2.2R9WivXe7CvWRVcGtF0BmIRAa1nGJthsDMLxBqmunE1oDYNLX6.HcnX7glDrV1LfgQ1UCg824.6LDpM2iIt1vQl1bdqj3bvIfbQQIWdvCTknoOnDF5W3fPnt2m7SCu6ZJiMMcqN7sNZ1kNJobpqWiyNIK8R0Yf0QaFGGC4RxhQN0AjacOY0.8ZeHfVk8yD-kZOwktTx7uBd9SqAA.AJP5ATlAd-v99ErN3-vj.XkKJUN8iX1f09zrxcN5uGiM1EbgqvFcjAe3U3-Pg-W6qI4lYmNaLAFrHWCHyaTec-IooxWFWqkCJ.l8ulGoVKi0PaF81OOtfuI.zxlFd.9tfjH5io2MStLZrCMwicImmtwifISJtdm9FwKrQNDimThC57acEFqy7TOxzY22yY1CkIMVUmLbhOhHF7N2n3.zjF3Y09pS3W06A5wHRvaLeoKtdJ.A9lBUndezhdQz0UP3APCuySt261DE8D505jJ78PgbiEsZXSs-zXZSGX03XP8fH599.BRTgMecIc26cjs20dNPQylic8Og9eYFls2p4BybsDp3OBVaKH7qo1IeZI6dx-wPPs5vwz4hx6TRAnKCgFsUOnUWyRcwY_&c=true&name=sqlite.md.txt). Very basic, but easily hackable.

**Comment by rhashimoto** [comment]
I'm not familiar with Starboard. It looks like it's similar to Jupyter but it runs within the browser? I finally figured out that I needed to open the browser console.

Is there a reason wa-sqlite is better or worse than sql.js here? Maybe it plays nicer with ES modules?

**Reply by twoxfh** [reply]
I could not figure out how to get sql.js to play nicely out of box which is where wa-sqlite came into play. A plugin might have smoother integration.  Starboard is like Jupyter Lite, all browser based.

**Reply by twoxfh** [reply]
For what its worth, I updated the example to be a plugin over at the starboard [issue](https://github.com/gzuidhof/starboard-notebook/issues/72). Its a bit more user friendly now.
